src/server/migrations/20201216124400_drop_open_feedback_from_feedback.js

```diff
@@ -0,0 +1,8 @@
+exports.up = function (knex) {
+  return knex.schema.table('feedback', function (table) {
+    table.dropColumn('open_feedback');
+  });
+};
+exports.down = function (knex) {
+  return knex.schema.table('feedback');
```

Comment:
You need to create column `open_feedback`

Reply:
The same as above.

Reply:
Noted and feedback implemted!